docs/src/further_topics/ugrid/operations.rst

```diff
@@ -488,7 +488,7 @@ earlier:
 
 
         # We'll re-use this to plot some real global data later.
-        >>> def cube_faces_to_polydata(cube):
+        >>> def cube_to_polydata(cube):
```

Comment:
Excuse a drive-by review, but shouldn't you now be replacing this definition with an import of the version from `iris.geovista` ?

Reply:
Patience please, we're working on it

Reply:
16ef5216c7c1de0a5f212978d968749a878cb8cc